compiler/src/iree/compiler/Codegen/Common/test/hoist_statically_bound_allocations.mlir

```diff
@@ -128,3 +128,26 @@ func.func @nested_op_alloca_yields(%arg0 : index, %arg1 : memref<?xi32>) -> memr
 //       CHECK:   scf.for
 //       CHECK:     %[[ALLOCA:.+]] = memref.alloca
 //       CHECK:     scf.yield %[[ALLOCA]]
+
+// -----
+
+#map = affine_map<(d0) -> (d0, 16)>
```

Comment:
Check that the `deallocs` get moved "down" as well?

Reply:
@MaheshRavishankar I've checked the dealloc op is moved down as well. Added a check for dealloc in the test.